KebabBackend/routes/api.php

```diff
@@ -70,8 +71,18 @@
 
     Route::middleware(['auth:sanctum', 'admin'])->post('/', [MeatTypeController::class, 'store']);
 
-    Route::middleware(['auth:sanctum', 'admin'])->put('{kebab}', [MeatTypeController::class, 'update']);
+    Route::middleware(['auth:sanctum', 'admin'])->put('{id}', [MeatTypeController::class, 'update']);
 
-    Route::middleware(['auth:sanctum', 'admin'])->delete('{kebab}', [MeatTypeController::class, 'destroy']);
+    Route::middleware(['auth:sanctum', 'admin'])->delete('{id}', [MeatTypeController::class, 'destroy']);
+});
+
+Route::prefix('saucetypes')->group(function () {
```

Comment:
> These paths have typos in them

I the typo is not in docs but in route 

```suggestion
Route::prefix('sauce-types')->group(function () {
```

Reply:
Perhaps it is better to change them to saucetypes to fit together with meattypes, now it's a bit inconsistent